### ISSUE TITLE
lets check your code for over fitting per this paper

### ISSUE THREAD

**Issue by Sandy4321** [dandxy89/DeepLearning_MachineLearning#1]
new drop out for embeddings layer : conventional drop outs leads to overfititng
http://arxiv.org/abs/1512.05287
A Theoretically Grounded Application of Dropout in Recurrent Neural Networks

Yarin Ga

**Comment by dandxy89** [comment]
@Sandy4321 did you follow up on this?

**Comment by Sandy4321** [comment]
No, it would be great to make it clear

**Comment by dandxy89** [comment]
Closing
